tcms/management/tests/test_admin.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import reverse
+
+from tcms.tests import LoggedInTestCase
+from tcms.tests.factories import BuildFactory, ComponentFactory
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestComponentAdmin(LoggedInTestCase):
```

Comment:
this is causing failures so something isn't right

Reply:
Fixed it! :smile: